src/components/Typography/index.tsx

```diff
@@ -22,7 +34,15 @@ export const H4: FC<TextProps> = (props) => {
 }
 
 export const H5: FC<TextProps> = (props) => {
-  return <Text as="h5" fontSize="2xl" lineHeight="32px" {...props} />
+  return (
+    <Text
+      as="h5"
+      fontWeight="500"
+      fontSize="24px"
+      lineHeight="32px"
+      {...props}
+    />
+  )
```

Comment:
Just flagging- in that case, we run into an issue where for smaller screens we have to use conditional rendering to change the font size. We need to develop a solution based on the `rem` units as I suggested here https://github.com/threshold-network/token-dashboard/pull/57#discussion_r790730467.